schema/gen/go/generate.go

```diff
@@ -1,14 +1,133 @@
+				case schema.StructRepresentation_Stringjoin:
+					fn(NewStructReprStringjoinGenerator(pkgName, t2, adjCfg), f)
+				default:
+					panic("unrecognized struct representation strategy")
+				}
+			case *schema.TypeMap:
+				fn(NewMapReprMapGenerator(pkgName, t2, adjCfg), f)
+			case *schema.TypeList:
+				fn(NewListReprListGenerator(pkgName, t2, adjCfg), f)
+			case *schema.TypeUnion:
+				switch t2.RepresentationStrategy().(type) {
+				case schema.UnionRepresentation_Keyed:
+					fn(NewUnionReprKeyedGenerator(pkgName, t2, adjCfg), f)
+				case schema.UnionRepresentation_Kinded:
+					fn(NewUnionReprKindedGenerator(pkgName, t2, adjCfg), f)
+				default:
+					panic("unrecognized union representation strategy")
+				}
+			default:
+				panic("add more type switches here :)")
+			}
+		}
+	}
+
+	// Emit a file with the type table, and the golang type defns for each type.
+	withFile(filepath.Join(pth, "ipldsch.types.gen.go"), func(f io.Writer) {
+		// Emit headers, import statements, etc.
+		fmt.Fprintf(f, "package %s\n\n", pkgName)
+		fmt.Fprintf(f, doNotEditComment+"\n\n")
+
+		// Emit the type table.
+		EmitTypeTable(pkgName, ts, adjCfg, f)
+
+		// Emit the type defns matching the schema types.
+		fmt.Fprintf(f, "\n// --- type definitions follow ---\n\n")
+		applyToEachType(func(tg TypeGenerator, w io.Writer) {
+			tg.EmitNativeType(w)
+			fmt.Fprintf(f, "\n")
+		}, f)
+
+	})
+
+	// Emit a file with all the Node/NodeBuilder/NodeAssembler boilerplate.
+	//  Also includes typedefs for representation-level data.
+	//  Also includes the MaybeT boilerplate.
+	withFile(filepath.Join(pth, "ipldsch.satisfaction.gen.go"), func(f io.Writer) {
+		// Emit headers, import statements, etc.
+		fmt.Fprintf(f, "package %s\n\n", pkgName)
+		fmt.Fprintf(f, doNotEditComment+"\n\n")
+		fmt.Fprintf(f, "import (\n")
+		fmt.Fprintf(f, "\tipld \"github.com/ipld/go-ipld-prime\"\n")        // referenced everywhere.
+		fmt.Fprintf(f, "\t\"github.com/ipld/go-ipld-prime/node/mixins\"\n") // referenced by node implementation guts.
+		fmt.Fprintf(f, "\t\"github.com/ipld/go-ipld-prime/schema\"\n")      // referenced by maybes (and surprisingly little else).
+		fmt.Fprintf(f, ")\n\n")
+
+		// For each type, we'll emit... everything except the native type, really.
+		applyToEachType(func(tg TypeGenerator, w io.Writer) {
+			tg.EmitNativeAccessors(w)
+			tg.EmitNativeBuilder(w)
+			tg.EmitNativeMaybe(w)
+			EmitNode(tg, w)
+			tg.EmitTypedNodeMethodType(w)
+			tg.EmitTypedNodeMethodRepresentation(w)
+
+			nrg := tg.GetRepresentationNodeGen()
+			EmitNode(nrg, w)
+
+			fmt.Fprintf(f, "\n")
+		}, f)
+	})
+}
+
+// GenerateSplayed is like Generate, but emits a differnet pattern of files.
+// GenerateSplayed emits many more individual files than Generate.
+//
+// This function should be considered deprecated and may be removed in the future.
+func GenerateSplayed(pth string, pkgName string, ts schema.TypeSystem, adjCfg *AdjunctCfg) {
```

Comment:
I'd  personally remove it directly. The old code will always be in the git history if we wish to reuse it.